src/test/java/com/google/sps/BooksTest.java

```diff
@@ -0,0 +1,54 @@
+package com.google.sps.servlets;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.net.URL;
+import org.mockito.Mockito;
+import java.io.IOException;
+import java.util.Map;
+import java.util.HashMap;
+
+/** */
+@RunWith(JUnit4.class)
+public final class BooksTest {
+  
+  public String EMPTY_MAP_STRING = "{}";
+  
+ /**
+  * Checks that if the keyword was nonsense, no result is returned.
```

Comment:
[Optional] I don't know if nonsense would be the correct word. There might be normal keywords that just don't return any results.